### ISSUE TITLE
Multidex support?

### ISSUE THREAD

**Issue by flankerhqd** [secure-software-engineering/CodeInspect#104]
Some apks put additional dex in apk's assets folder or somewhere else. I wonder if the following steps are enough to enable debugging into these addtional dex?

- manually extracting these dex
- transforming them into jimple
- merge into CodeInpsect project source folder

**Comment by StevenArzt** [comment]
@MarcMil There should be a function to merge a dex file into the current project. Is this already implemented?

**Comment by MarcMil** [comment]
Yes, it is. However, I would recommend using the newest nightly build for this, since the process has been further simplified.

If the dex file is within your project, you can use the popup menu on the dex file and click on "Merge dex file into this project". Alternatively, you can use the popup menu on the project itself and click on "Merge dex file" and select the dex file from disk.

**Comment by flankerhqd** [comment]
However, I mean if dex methods count exceeds the limit 65536, which is the main reason some vendors split dex into multiple files, can CodeInspect properly handle this situation? It seems apktool/smali cannot merge the multiple dex into one single file because of that limit.

**Comment by StevenArzt** [comment]
We do not support this yet, this should be implemented at some point. That's why I re-open the issue.
